onebusaway-android/src/main/res/layout/trip_details_head.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (C) 2010-2015 Paul Watts (paulcwatts@gmail.com)
+    and individual contributors.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<TableLayout xmlns:android="http://schemas.android.com/apk/res/android"
+             style="@style/HeaderItem"
+             android:layout_width="fill_parent"
+             android:layout_height="wrap_content"
+             android:orientation="vertical">
+    <TableRow>
+        <TextView
+                android:id="@+id/short_name"
+                style="@style/RouteInfoShortName"
+                android:layout_width="fill_parent"
+                android:layout_height="wrap_content"
+                android:layout_column="0"
+                android:singleLine="false">
+        </TextView>
+        <LinearLayout
+                android:orientation="vertical"
+                android:layout_gravity="center_vertical"
+                android:layout_column="1"
+                android:layout_weight="1">
+            <TextView
+                    android:id="@+id/long_name"
+                    style="@style/RouteInfoLongName"
+                    android:layout_width="fill_parent"
+                    android:layout_height="wrap_content">
+            </TextView>
+            <LinearLayout android:orientation="horizontal"
+                          android:layout_gravity="left"
+                          android:layout_width="fill_parent"
+                          android:layout_height="wrap_content">
+                <TextView
+                        android:id="@+id/agency"
+                        style="@style/TripDetailsAgency"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content">
+                </TextView>
+                <TextView
```

Comment:
Can you please move the `vehicle` TextView to the following line, on its own line?  This gets really cramped for agencies that have longer names, and also use their name as the agency_id that prefaces the vehicle number:

![image](https://cloud.githubusercontent.com/assets/928045/10032183/8a59df4a-614f-11e5-9676-59e4d2d991c5.png)